contracts/Identity.sol

```diff
@@ -58,7 +58,7 @@ contract Identity is KeyManager {
   returns (bool valid)
   {
     bytes32 pbKey = addressToKey(
-      _toSign.toEthSignedMessageHash().recover(_signature)
+      _message.toEthSignedMessageHash().recover(_signature)
     );
 
```

Comment:
I think it would be a bit easier to understand in the following way:
Logical OR's and negotiation or often a bit harder to understand. (but just a personal opinon)

```javascript
 if (keyHasPurpose(pbKey, _purpose) &&  keys[pbKey].revokedAt == 0) {
    return true;
}
return false;


```